tests/integration/suite/test_system_project.py

```diff
@@ -11,6 +11,7 @@
                                  "kube-public",
                                  "cattle-global-data",
                                  "cattle-global-nt",
+                                 "cattle-impersonation-system",
```

Comment:
@cmurphy Maybe can there be a timing issue involved?
- Do we create this "cattle-impersonation-system" namespace during startup always? If yes then this change is needed in 2.6 also?
- Or if it depends on some other usecase to get created, are there chances that some testcase created it? Example this test handles the logging-namespace and removes it from the comparison.

Reply:
I'm investigating. It looks like this test is run on the local cluster and the namespace shouldn't actually be created there, so we shouldn't need this.